proxy/src/auth/backend.rs

```diff
@@ -288,45 +364,50 @@ impl BackendType<'_, ClientCredentials<'_>> {
                     "performing authentication using the console"
                 );
 
-                let api = api.as_ref();
-                auth_quirks(
-                    api,
+                let success = auth_quirks(
```

Comment:
nit(here and bellow): I would unpack it `let (cache_info, user_info) = auth_quirks(...`.